lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -314,16 +325,28 @@ describe('CategoryRenderer', () => {
     });
   });
 
-  describe('clumping passed/failed/manual', () => {
+  describe('clumping passed/failed/warning/manual', () => {
```

Comment:
will you also add another test that has a warning on a passed audit and one on a failed audit and only the passed audit shows up in the warning section?